plugins/outputs/influxdb/http.go

```diff
@@ -173,6 +175,7 @@ func NewHTTPClient(config HTTPConfig) (*httpClient, error) {
 		},
 		createdDatabases: make(map[string]bool),
 		config:           config,
+		log:              models.Logger{Name: userAgent},
```

Comment:
Don't instantiate Logger here.  Maybe some leftover code from development?

Reply:
Not leftover, the httpclient logs things, so this was intentionally added so it could.